internal/pkg/plugininstaller/reposcaffolding/option.go

```diff
@@ -10,6 +10,7 @@ import (
 const (
 	transitBranch      = "init-with-devstream"
 	appNamePlaceHolder = "_app_name_"
+	defaultCommitMsg   = "init with devstream"
 )
 
 type Options struct {
```

Comment:
`required` does not make sense for non-pointer structs. Remove `required` or change to `*STRUCTNAME`